apps/dashboard/public/noVNC-1.1.0/vnc.html

```diff
@@ -76,7 +76,7 @@
             }
         });
     </script>
-    <script type="module" crossorigin="use-credentials" src="app/ui.js"></script>
+    <script type="module" crossorigin="anonymous" src="app/ui.js"></script>
```

Comment:
```suggestion
    <script type="module" crossorigin="use-credentials" src="app/ui.js"></script>
```

This has to be `use-credentials` because Safari throws a CORS errors, see https://github.com/OSC/ondemand/pull/516